Lib/crypt.py

```diff
@@ -19,18 +19,31 @@ def __repr__(self):
         return '<crypt.METHOD_{}>'.format(self.name)
 
 
-def mksalt(method=None, *, log_rounds=12):
+def mksalt(method=None, *, rounds=None):
     """Generate a salt for the specified method.
 
     If not specified, the strongest available method will be used.
 
     """
     if method is None:
         method = methods[0]
-    if not method.ident:
+    if not method.ident:  # traditional
         s = ''
```

Comment:
I suggest to raise a ValueError if rounds is set (is not None) but the method doesn't accept a number of rounds.